src/ResourceManager/KeyVault/Commands.KeyVault/Commands/SetAzureKeyVaultAccessPolicy.cs

```diff
@@ -198,6 +212,24 @@ public class SetAzureKeyVaultAccessPolicy : KeyVaultManagementCmdletBase
         [ValidateSet("get", "list", "delete", "create", "import", "update", "managecontacts", "getissuers", "listissuers", "setissuers", "deleteissuers", "manageissuers", "all")]
         public string[] PermissionsToCertificates { get; set; }
 
+        /// <summary>
+        /// Permissions to Storage
+        /// </summary>
+        [Parameter( Mandatory = false,
+            ParameterSetName = ByObjectId,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Specifies managed storage account and sas definition operation permissions to grant to a user or service principal." )]
+        [Parameter( Mandatory = false,
+            ParameterSetName = ByServicePrincipalName,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Specifies managed storage account and sas definition operation permissions to grant to a user or service principal." )]
+        [Parameter( Mandatory = false,
+            ParameterSetName = ByUserPrincipalName,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Specifies managed storage account and sas definition operation permissions to grant to a user or service principal." )]
+        [ValidateSet( "get", "list", "delete", "set", "update", "regeneratekey", "getsas", "listsas", "deletesas", "setsas", "all" )]
+        public string[] PermissionsToStorage { get; set; }
```

Comment:
This reflects existing naming pattern (PermissionsToSecrets/Keys/Certificates) and will be retained for consistency.